src/ucp/core/ucp_request.h

```diff
@@ -183,6 +190,8 @@ struct ucp_request {
                 uct_completion_t  uct_comp; /* UCT completion */
             } state;
 
+            ucp_lane_index_t      pend_add_lane; /* Lane on which request was moved
```

Comment:
let's rename this field to just 'pending_lane'

Reply:
renamed